lib/gnat.ex

```diff
@@ -152,49 +158,70 @@ defmodule Gnat do
   """
   def ping(pid) do
     GenServer.call(pid, {:ping, self()})
+
     receive do
       :pong -> :ok
     after
-      3_000 -> {:error, "No PONG response after 3 sec"}
+      3000 -> {:error, "No PONG response after 3 sec"}
     end
   end
 
   @impl GenServer
   def init(connection_settings) do
     connection_settings = Map.merge(@default_connection_settings, connection_settings)
+
     case Gnat.Handshake.connect(connection_settings) do
       {:ok, socket} ->
-        parser = Parser.new
-        {:ok, %{socket: socket, connection_settings: connection_settings, next_sid: 1, receivers: %{}, parser: parser}}
+        parser = Parser.new()
+
+        {
+          :ok,
+          %{
+            socket: socket,
+            connection_settings: connection_settings,
+            next_sid: 1,
+            receivers: %{},
+            parser: parser
+          }
+        }
```

Comment:
Similar here. Formatter is trying to avoid long lines, but so much space in this tuple is odd to me. Maybe instead?
```elixir
response = %{socket: socket, connection_settings: connection_settings, next_sid: 1, receivers: %{}, parser: parser}
{:ok, response}
```

Reply:
I actually like this format rule.  I'm willing to trade the extra whitespace to make the map significantly more readable.